src/openrct2-ui/windows/Map.cpp

```diff
@@ -101,7 +101,7 @@ static rct_widget window_map_widgets[] = {
 
 // used in transforming viewport view coordinates to minimap coordinates
 // rct2: 0x00981BBC
-static constexpr const LocationXY16 MiniMapOffsets[] = {
+static constexpr const CoordsXY MiniMapOffsets[] = {
```

Comment:
The idea of the Coords and TileCoords structures is that you can be sure about what's stored inside. 

These are not big coords or small coords, and thus shouldn't use CoordsXY or TileCoordsXY.

Reply:
I've reverted these changes.